docs/pages/tools/integration/pentaho.md

```diff
@@ -1,112 +1,142 @@
-*Last Update {{ git_revision_date }}* 
+---
+tags: 'integration, middleware, python, IoT'
+icon: 'https://awesome-astra.github.io/docs/img/pentaho-data-integration/logo-pentaho.png'
+developer_title: 'Hitachi'
+developer_url: 'https://sourceforge.net/projects/pentaho/'
+links:
```

Comment:
@gstachni -- I took a stab at the links; please confirm when you're able.

Reply:
Very cool